keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -1001,7 +1124,66 @@ _This section must be completed when targeting beta graduation to a release._
 
 * **What are other known failure modes?**
   For each of them, fill in the following information by copying the below template:
-  N/A
+
+<!--
+For each of them, fill in the following information by copying the below template:
+  - [Failure mode brief description]
+    - Detection: How can it be detected via metrics? Stated another way:
+      how can an operator troubleshoot without logging into a master or worker node?
+    - Mitigations: What can be done to stop the bleeding, especially for already
+      running user workloads?
+    - Diagnostics: What are the useful log messages and their required logging
+      levels that could help debug the issue?
+      Not required until feature graduated to beta.
+    - Testing: Are there any tests for failure mode? If not, describe why.
+-->
+
+    - [InClusterConfig() fails inside HostProcessContainers]
+      - Causes: 
+        - Due to how volume mounts are configured in containerd v1.6+ service account tokens in the container are
+          present at the location expected by the golang API rest client.
+      - Mitigations: 
+        - If containers are using `symlink` mount behavior as described at [Compatibility](#compatibility) you can construct a kubeconfig
+          file with the containers assigned service account token and use that to authenticate.
+          Example: https://github.com/kubernetes-sigs/sig-windows-tools/blob/fbe00b42e2a5cca06bc182e1b6ee579bd65ed1b5/hostprocess/calico/install/calico-install.ps1#L8-L11
+        - Switch to container runtime/version that supports `bind` mount behavior as as described at [Compatibility](#compatibility)
+      - Diagnostics:
+        - Calls to rest.InClusterConfig will fail in workloads.
+      - Testing:
+        - No - known limitation
+
+    - [Containers running as non-HostProcessContainers]
+      - Causes:
+        - Container runtime does not support HostProcessContainers
+        - Bug in kubelet in some v1.23/v1.24 patch versions [#110140](https://github.com/kubernetes/kubernetes/pull/110140)
+      - Detection:
+        - Varries based on cause
+        - Likely result will be ContainerCreate failures of some kind
```

Comment:
Would this cause the metric `started_host_process_containers_errors_total` to be updated?

Reply:
This really varies by situation but in most cases I do not think `started_host_process_containers_errors_total` will be incremented.
In cases like this the containers would be created successfully but apps running inside those containers would fail because files they are trying access files that are in a different location or do perform operations that cannot be performed in non-hostprocess containers.

Reply:
I'll add clarification in the KEP that these will likely show up as app/workload errors

Reply:
updated